CMakeLists.txt

```diff
@@ -180,6 +184,24 @@ mark_as_advanced(BUILD_FOR_PACKAGING)
 cmake_dependent_option(FORCE_LEGACY_LIBBPF "Force usage of libbpf 0.0.9 instead of the latest version." False "ENABLE_PLUGIN_EBPF" False)
 mark_as_advanced(FORCE_LEGACY_LIBBPF)
 
+# checks link with cmake required libs
+cmake_policy(SET CMP0075 NEW)
```

Comment:
This can actually be dropped, the relevant policy is implicitly set by our minimum required CMake version.